docs/publication.md

```diff
@@ -0,0 +1,5 @@
+# Metrics publication
```

Comment:
this section would be the "Experiment tracking" that can include publication of metrics and other things like logs, configs etc. and details on integration with Taskcluster